schema/v2.yaml

```diff
@@ -407,6 +407,15 @@ models:
       optional: true
       nullable: false
       default: false
+    - field: isUnread
+      description: |
+        Whether the conversation was explicitly marked as unread.
+
+        Note that the `unreadCount` always takes precedence over this flag.
```

Comment:
That's misleading. IIRC they are treated equally.

Reply:
If a conversation is marked as unread without having unread messages, it is marked with an empty "unread message count" bubble, not with a zero. This is what I was trying to say: If both `isUnread` is set and `unreadCount` is 3, then the bubble should contain the number 3, and not be empty. Display logic from `unreadCount` takes precedence over display logic for `isUnread`.

What about this?

> Note that a non-zero `unreadCount` always takes precedence over this flag.

Reply:
:+1: